### DISCUSSION TITLE
Impossible to load from HA

### DISCUSSION THREAD

**Discussion by bzh49** [hassio-addons/addon-tasmoadmin#222] [Q&A]
It Seems there is a bug with links to download add-on from HA because files seems unknown 

Can't install ghcr.io/hassio-addons/tasmoadmin/aarch64:0.16.0: 404 Client Error for http+docker://localhost/v1.41/images/ghcr.io/hassio-addons/tasmoadmin/aarch64:0.16.0/json: Not Found ("no such image: ghcr.io/hassio-addons/tasmoadmin/aarch64:0.16.0: No such image: ghcr.io/hassio-addons/tasmoadmin/aarch64:0.16.0")

Any clues to get the add-on ?

**Comment by frenck** [comment]
Seems to be fine and exists: <https://github.com/hassio-addons/addon-tasmoadmin/pkgs/container/tasmoadmin%2Faarch64/10134274?tag=0.16.0>

You can check it yourself, by pasting the ghcr.io link from the error message into your browser.

Please note, that Docker throws a 404 for many reasons. One, for example, is disk space-related, or it can actually be a resolution issue on your network's end.
